alpine-base-s6/Dockerfile.tmpl

```diff
@@ -25,7 +25,7 @@ LABEL org.label-schema.schema-version="1.0" \
       org.label-schema.distribution-version=$VERSION \
       org.label-schema.s6-version=$S6_VERSION
 
-RUN curl -sL https://github.com/just-containers/s6-overlay/releases/download/v${S6_VERSION}/s6-overlay-amd64.tar.gz -o /tmp/s6.tgz && \
+RUN curl -sL https://github.com/just-containers/s6-overlay/releases/download/${S6_VERSION}/s6-overlay-amd64.tar.gz -o /tmp/s6.tgz && \
```

Comment:
The version number needs the 'v' back in front.

Reply:
I removed the "v" just to add it to the actual version. if you `docker inspect` a container created from alpine-base-s6 latest tag, you should see the S6 version includes the "v" now.